Directory.Build.props

```diff
@@ -4,6 +4,10 @@
     <LangVersion>preview</LangVersion>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <AnalysisLevel>latest</AnalysisLevel>
+    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+    <NoWarn>CA1031;CA1711</NoWarn>
```

Comment:
Can we _no_ disable these warnings globally?

Reply:
`Exception boundary.` is fine. ;P

Reply:
That solves `CA1031` but what about `CA1711`?

Reply:
I don't find a warning that my _test collection_ class has name ending with `Collection` reasonable enough to keep.